modules/combined/test/tests/reference_residual/tests

```diff
@@ -5,6 +5,9 @@
     exodiff = 'reference_residual_out.e'
     custom_cmp = 'reference_residual.cmp'
     scale_refine = 2
+    design = 'ReferenceResidualProblem.md'
+    issues = 'r22364'
```

Comment:
@aeslaughter how should I handle issues for commits made with an svn number?

Reply:
I usually use whatever issue number created the tests file, if I can find it:  git log --reverse tests

#12053

Reply:
Thanks!